pkg/operator2/operator.go

```diff
@@ -10,6 +10,9 @@ import (
 	"strconv"
 	"strings"
 
+	"monis.app/go/openshift/controller"
+	"monis.app/go/openshift/operator"
```

Comment:
Hmm, @enj do these need to get moved back into an OpenShift repo (library-go?) in the wake of #205?

Reply:
https://github.com/openshift/operator-boilerplate-legacy